src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountCollection.java

```diff
@@ -64,12 +65,34 @@ public void add(final SimpleInterval interval, final int refReadCount, final int
         counts.add(new AllelicCount(interval, refReadCount, altReadCount));
     }
 
+    /**
+     * Adds (interval, reference count, alternate count, reference nucleotide, alternate nucleotide) to respective lists.
+     * @param interval      site in 1-based format
+     * @param refReadCount  number of reads at site matching the reference
+     * @param altReadCount  number of reads at site different from the reference
+     * @param refNucleotide the ref nucleotide
+     * @param altNucleotide the alt nucleotide
+     */
+    public void add(final SimpleInterval interval, final int refReadCount, final int altReadCount,
```

Comment:
I agree with your comment above: it is a good idea to decouple. I removed the explicit `add` functions in favor of just one that takes `AllelicCount`. Updated the dependent classes as well.